packages/core/src/writers/single-mode.ts

```diff
@@ -115,6 +115,10 @@ export const writeSingleMode = async ({
       data += getOrvalGeneratedTypes();
       data += '\n';
     }
+    if (implementation.includes('<HTTPStatusCodes,')) {
```

Comment:
I don't want to include the processing of `fetch`client only in the processing of each core writer. Could you move these operations into the `fetch` package to confine their scope of influence?